### PR TITLE
Updating push landing page

### PR DESCRIPTION
**Pull request by matzew**
@balunasj here are the changes you requested, to make UPS more obvious, instead of starting w/ SPS/WPS servers.

### CODE REVIEW

**Comment by matzew** [comment]
@sebastienblanc @danbev can you review ? 

Perhaps a review from @lfryc and/or @andresgalante is good as well, for planing of the 'details' page for the modules, that we do on the new homepage

File: push/index.markdown [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: basic
+title: AeroGear Push
+---
+
+# AeroGear Push
+
+AeroGear has support for different push notification technologies:
+
+* [UnifiedPush Server](#unifiedpush-server)
```
Comment (lfryc) [inline push/index.markdown]: Perhaps we may note the platform that Unified Push support in a parenthesis:

* UnifiedPush (supports Android, iOS, Windows, FirefoxOS)

I know it is noted right bellow, but still may help people navigate easily - sometimes you just want "Push to Android", and term UnifiedPush is far from your thinking

**Comment by matzew** [comment]
@lfryc good idea

**Comment by andresgalante** [comment]
The only issue I have with this is that using markdown we can't use btns with icons as I did on https://github.com/aerogear/aerogear.org/pull/476/files
If that is not a problem, then changing the Front matter will do the trick for the new design.

**Comment by matzew** [comment]
@andresgalante this is just a temporary solution. on the new website, we can go your suggested way - but for the old one, this is just something, to better present the UPS.

**Comment by andresgalante** [comment]
Perfect then :+1:

**Comment by danielpassos** [comment]
:+1:

**Comment by cvasilak** [comment]
:rocket: landed on https://github.com/aerogear/aerogear.org/commit/7d9fa3aa9d19e7c0faddb6fa01c3140ecb393f7d

**Pull request closed without merging**